src/MetaMorpho.sol

```diff
@@ -799,6 +827,26 @@ contract MetaMorpho is ERC4626, ERC20Permit, Ownable2Step, Multicall, IMetaMorph
         return UtilsLib.min(supplyShares.toAssetsDown(totalSupplyAssets, totalSupplyShares), availableLiquidity);
     }
 
+    /// @dev Returns the withdrawable amount of assets from the market defined by `marketParams`.
+    /// @dev Assumes that the inputs `marketParams` and `id` match.
+    /// @dev Must not revert.
+    function _simulateWithdrawable(MarketParams memory marketParams, Id id) internal view returns (uint256) {
```

Comment:
Shouldn't we just replace `_withdrawable` with this ?

Reply:
It costs more gas though so I'm not sure.

But we could argue that it removes a function